pinot-plugins/pinot-input-format/pinot-avro-base/src/main/java/org/apache/pinot/plugin/inputformat/avro/AvroSchemaUtil.java

```diff
@@ -95,6 +100,9 @@ public static ObjectNode toAvroSchemaJsonObject(FieldSpec fieldSpec) {
       case BYTES:
         jsonSchema.set("type", convertStringsToJsonArray("null", "bytes"));
         return jsonSchema;
+      case JSON:
+        jsonSchema.set("type", convertStringsToJsonArray("null", "json"));
```

Comment:
Should this be `string`? I don't think avro has `json` type

Reply:
Done